.travis.yml

```diff
@@ -79,6 +80,9 @@ matrix:
       - dist: trusty
         env:
           - JOB="2.7_SLOW" SLOW=true
+      - dist: trusty
+        env:
+        - JOB="3.6_CONDA_BUILD_TEST" TEST_ARGS="--skip-slow" BUILD_TEST=true
```

Comment:
need a space here, this line has to EXACTLY match the one above. add back --skip-network to TEST_ARGS

Reply:
don't add this to the allowed failures, it should be a main build test only

Reply:
Hi Jeff, should I remove BUILD_TEST(soon to be PIP_BUILD_TEST) from allowed failures as well?